09LessonNine/in-class-project/css/style.css

```diff
@@ -0,0 +1,106 @@
+h1, h4, p, a{
+  font-family: 'Montserrat', serif;
+}
+
+h1{
+  display: block;
+  font-size: 2em;
+  font-weight: bold;
+}
+
+h4{
+  display: block;
+  font-size: 1em;
+  font-weight: bold;
+}
+
+.panel{
+  color: white;
+  margin: 1vh 1vw;
+  position: relative;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.panel div:not(.opacity){
+  position: relative;
+  margin: 0px 30px;
+  top: 30%;
+}
+
+.half-col, .half-row, .quarter{
+  vertical-align: top;
+}
+
+.full{
+  width: 100vw;
+  height: 100vh;
+  margin: 0vh 0vw;
+}
+
+.half-col{
+  display: inline-block;
+  width: 48vw;
+  height: 98vh;
+}
+
+.half-row{
+  display: inline-block;
+  width: 98vw;
+  height: 48vh;
+}
+
+
+.quarter{
+  display: inline-block;
+  width: 48vw;
+  height: 48vh;
+}
+
+.opacity{
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+.title{
+  margin-bottom: 15px;
+}
+
+.subtitle{
+  color: #C0D890;
+  margin-bottom: 15px;
+}
+
+.description{
+  margin-bottom: 50px;
+}
+
+.button{
+  background-color: green;
+  text-decoration: none;
+  margin: 0px 2% 0px 0px;
+  padding: 10px 15px;
+  color: white;
+}
+
+.text-center{
+  text-align: center;
+}
+
```

Comment:
your services page is suffering from non-uniform margins and widths. the proportion of the two left boxes and the one on the right is off as well. check out madeline's here:  http://madelineolsen.github.io/intro-workbook/09LessonNine/in-class-project/services.html